gulpfile.js

```diff
@@ -825,17 +826,32 @@ function checkBinarySize(compiled) {
   const file = compiled ? './dist/v0.js' : './dist/amp.js';
   const size = compiled ? '77.05KB' : '336.66KB';
   const cmd = `npx bundlesize -f "${file}" -s "${size}"`;
```

Comment:
Any reason to use `npx` here instead of `yarn`?

Reply:
The output of `yarn bundlesize` contains even more extra noise than `npx bundlesize`.

**`yarn`:**
```
~/src/amphtml$ yarn bundlesize -f "./dist/amp.js" -s "336.37 kB"
yarn run v1.6.0
$ /usr/local/google/home/rsimha/src/amphtml/node_modules/.bin/bundlesize -f ./dist/amp.js -s '336.37 kB'
 WARN  github token not found

    You are missing out on some cool features.
    Read more here: https://github.com/siddharthkp/bundlesize#2-build-status
   

 PASS  ./dist/amp.js: 336.37KB < maxSize 336.37KB (gzip) 

Done in 0.45s.
```
**`npx`**:
```
~/src/amphtml$ npx bundlesize -f "./dist/amp.js" -s "336.37 kB"
 WARN  github token not found

    You are missing out on some cool features.
    Read more here: https://github.com/siddharthkp/bundlesize#2-build-status
   

 PASS  ./dist/amp.js: 336.37KB < maxSize 336.37KB (gzip) 
```